src/id3_variants_training/__main__.py

```diff
@@ -0,0 +1,32 @@
+from .ConfusionMatrix import ConfusionMatrix
+import argparse
+from pkg_resources import resource_filename
+import config
+
+
+def main():
+    default_file_path = resource_filename(config.__name__, 'config.json')
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--config_file', help='Path to the config file that contains variant ranges in JSON format',
+                        default=default_file_path)
+    parser.add_argument('--use_server_vcf_files', action='store_true', default=False,
```

Comment:
This is good but I think the name could be changed; training on the remote data, it won't be getting VCF files from the server but making queries about individual variants.  Maybe something like "--use-candig-apis" or something?

Reply:
Yeah, I think `--use-candig-apis` makes more sense. What should I change the help text to for this argument then?

Reply:
"Use remote API access to variant information rather than local VCF files"